scripts/qa_crowbarsetup.sh

```diff
@@ -622,6 +622,8 @@ function onadmin_prepare_cloud_repos()
             develcloud5)
                 addsp3testupdates
                 add_sles12ga_testupdates
+                addcloud5maintupdates
```

Comment:
I'm not sure when this would happen, given that D:C:5:S would have had more builds than the "official" builds. So the build numbers should always be higher, afaict.